### PR TITLE
Spike customer journey app

### PR DESCRIPTION
**Pull request by pjbull**
- Prototype of customer journey app for interacting with the mobile money ladder.
 - Notebook is just for prototyping, runable code is in `src`
 - Add to makefile
 - BONUS: remove outdated instructions from README and Makefile

### CODE REVIEW

**Comment by pjbull** [comment]
@ejm714 FYI, this can wait until after our MMR check-in

File: src/visualization/bokeh_mm_ladder/main.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+def _refresh_data(data, grouper, pct_of):
+    """ Select and adjust data based on grouping by `grouper`
+        and calculating prevalence based on entire population
+        or just for the respective mobile money ladder level.
+    """
+    data[grouper] = data[grouper].astype(str)
+
+    if pct_of == 'Total Population':
+        to_plot = df.groupby(['mm_ladder'])[grouper].value_counts().pipe(lambda x: x / x.sum())
```
Comment (ejm714) [inline src/visualization/bokeh_mm_ladder/main.py]: should be `data.groupby` not df

File: src/visualization/bokeh_mm_ladder/main.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    else:
+        to_plot = df.groupby(['mm_ladder'])[grouper].value_counts(normalize=True)
```
Comment (ejm714) [inline src/visualization/bokeh_mm_ladder/main.py]: should be `data.groupby`

**Comment by pjbull** [comment]
@ejm714 Thanks, good catch!

**Pull request merged**